include/ff/parallel_execution_ff.h

```diff
@@ -79,8 +79,9 @@ class parallel_execution_ff {
         a lower value.
   */
   void set_concurrency_degree(int degree) noexcept { 
-    concurrency_degree_ = std::min(degree,
-        static_cast<int>(std::thread::hardware_concurrency())); 
+//    concurrency_degree_ = std::min(degree,
+//        static_cast<int>(std::thread::hardware_concurrency()));
```

Comment:
If you are removing the commented lines, please remove them (not just comment out).

Reply:
fixed